modules/lvms-monitoring-openshift-data-foundation-logical-volume-manager-operator.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// storage/logical-volume-manager-storage.adoc
+
+:_content-type: PROCEDURE
+[id="lvms-monitoring-using-lvms_{context}"]
+= Monitoring the logical volume manager storage
+
+When the logical volume manager storage is installed using the OpenShift Web Console, you can monitor the cluster by using the `Block and File` dashboard in the console by default. However, when you use {rh-rhacm} to install the logical volume manager storage, you need to configure the {rh-rhacm} Observability to monitor all the {sno} clusters from one place.
+
+== Metrics
+
+You can monitor the logical volume manager storage by viewing the metrics exported by the Operator on the {rh-rhacm} dashboards and the alerts that are triggered.
+
+* Add the following `topolvm` metrics to the `allow` list:
++
+[source,terminal]
+----
+topolvm_thinpool_data_percent
+topolvm_thinpool_metadata_percent
+topolvm_thinpool_size_bytes
+----
+
+[NOTE]
+====
+Metrics are updated every 10 minutes or when there is a change in the thin pool, such as a new logical volume creation.
+====
+
+== Alerts
+
+When the thin pool and volume group are filled up, further operations fail and might lead to data loss.
+The logical volume manager storage sends the following alerts about the usage of the thin pool and volume group when utilization crosses a certain value:
```

Comment:
The original sentence seemed to be missing something. Does this make sense?
Original sentence:
> The logical volume manager storage sends the following alerts the usage of the thin pool and volume group crosses certain value:

Reply:
That makes sense to me. The only adjustment would be to drop the preceding `the`.

Reply:
Done and dropped `the` in all of the files for logical volume manager storage.